distribution/packages/src/deb/debian/preinst

```diff
@@ -11,49 +11,66 @@
 
 set -e
 
-# Reference to temp directory
-tmp_dir=/var/lib/wazuh-indexer/tmp
-restart_service=${tmp_dir}/wazuh-indexer.restart
+name="wazuh-indexer"
+product_dir=/usr/share/${name}
+config_dir=/etc/${name}
+data_dir=/var/lib/${name}
+log_dir=/var/log/${name}
+pid_dir=/run/${name}
+tmp_dir=${data_dir}/tmp
+state_file=${config_dir}/.was_active
 
-# Create needed directories
+# Cleanup temp directory
 if [ -d ${tmp_dir} ]; then
     rm -r ${tmp_dir}
 fi
+# Create needed directories
 mkdir -p ${tmp_dir}
```

Comment:
We can safely remove this code and any other code in charge of managing the `tmp_dir`, as it won't be supported on our wazuh-indexer@5.0.0 packages and these will not be compatible with 4.x.